core/src/main/scala/kafka/common/ErrorMapping.scala

```diff
@@ -62,6 +62,7 @@ object ErrorMapping {
   val TopicAuthorizationCode: Short = 29
   val GroupAuthorizationCode: Short = 30
   val ClusterAuthorizationCode: Short = 31
+  // 32: INVALID_TIMESTAMP
```

Comment:
`ConsumerFetcherThread` and `SimpleConsumer` still use this class. Given that, are we sure that we don't need to add a mapping here?